bin/hdfcoinc/pycbc_coinc_hdfinjfind

```diff
@@ -206,14 +206,14 @@ for trigger_file, injection_file in zip(args.trigger_files,
                    sim_table.get_column(args.redshift_column))
 
     if multi_ifo_style:
-        for key in f.keys():
+        for key in f['segments'].keys():
             if key == 'foreground':
                 continue
```

Comment:
Is this bit needed now?

Reply:
yes we do, see line 161-165 of pycbc_multiifo_coinc_statmap